cheeks.css

```diff
@@ -0,0 +1,27 @@
+body{
+    font-family: monospace;
+    background-color: lavender;
+    display: flex;
+    flex-direction: column;
+    align-items: center ;
+}
+header{
```

Comment:
we could use the align-items property to align the text in the center via y-axis